src/main/java/org/candlepin/subscriptions/tally/job/CaptureSnapshotsTaskManager.java

```diff
@@ -137,13 +137,15 @@ public void updateHourlySnapshotsForAllAccounts() {
 
             Duration metricRange = appProperties.getMetricLookupRangeDuration();
             Duration prometheusLatencyDuration = appProperties.getPrometheusLatencyDuration();
+            Duration hourlyTallyOffsetMinutes = appProperties.getHourlyTallyOffsetMinutes();
 
-            OffsetDateTime endTime = adjustTimeForLatency(OffsetDateTime.now().truncatedTo(ChronoUnit.HOURS),
-                prometheusLatencyDuration);
-            OffsetDateTime startTime = endTime.minus(metricRange);
+            OffsetDateTime endDateTime = adjustTimeForLatency(OffsetDateTime.now()
+                .minus(hourlyTallyOffsetMinutes).truncatedTo(ChronoUnit.HOURS), prometheusLatencyDuration);
+
+            OffsetDateTime startDateTime = endDateTime.minus(metricRange).minus(hourlyTallyOffsetMinutes);
```

Comment:
@dmckoske found one more tiny thing - since the end date time already has the offset applied, don't need to apply again to the `startTime`.

```suggestion
            OffsetDateTime startDateTime = endDateTime.minus(metricRange);
```

FWIW, I've been looking at the output of `HOURLY_TALLY_OFFSET=3h SPRING_PROFILES_ACTIVE=capture-snapshots ./gradlew bootRun` to verify.

Reply:
Yeah, sorry about that.  I see the calculation now.